src/components/molecules/NavList.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box } from '@mantine/core';
+
+import { DashboardIcon } from '../appShell/icons/DashboardIcon';
+import { PackageIcon } from '../appShell/icons/PackageIcon';
+import { PeopleIcon } from '../appShell/icons/PeopleIcon';
+import { NavLink } from '../atoms/NavLink';
+
+export const NavList = () => {
+  return (
+    <Box sx={{ width: 230 }}>
+      <NavLink
+        links={[
+          {
+            label: 'داشبورد',
+            icon: <DashboardIcon w="24" h="24" />,
+          },
+          {
+            label: 'خانواده ها',
+            icon: <PeopleIcon w="24" h="24" />,
+          },
+          {
+            label: 'پروژه ها',
+            icon: <PackageIcon w="24" h="24" />,
+          },
+        ]}
+      />
+    </Box>
+  );
+};
```

Comment:
we don't really need this abstraction we can just put it directly into app shell

Reply:
no that's what I'm saying, we avoid passing anything to app shell cause we don't need it anywhere else

the benefit, well is that we avoid leaky abstractions, the real question isn't why we should't abstract it, it's should we abstract it or not, which the answer is no we don't need to abstract it cause it adds no benefit

Reply:
Oh sorry now I realized I have had the wrong idea about app shell (thinking it's the same thing as sidebar)
forget everything I said and sorry for taking your time